content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
 
 ## Overview
 
-In general, [custom metrics][1] refers to any metrics that are not part of the regular integration suite, for example using custom checks or API-level metrics in your application stack.
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a [custom metric][2]<sup>[(1)](#standard-integrations)</sup>.
 
-* Billable count is based on average number of `custom metrics / hour` for the month.
-* Pro plans include 100 custom metrics per host.
-* Enterprise plans include 200 custom metrics per host.
-* The number of metrics is averaged across all paid hosts.
-* Additional custom metrics packages can be purchased.
+## Allocation
 
-Contact [Sales][2] or your [Customer Success][3] Manager to discuss custom metrics for your account.
+You are allocated a certain number of custom metrics based on your Datadog pricing plan:
+
+* Pro: 100 custom metrics per host.
+* Enterprise: 200 custom metrics per host.
+
+The billable number of custom metrics is based on the average number of custom metrics (from all paid hosts) per hour over a given month.
```

Comment:
Let's move the sentence on the billable number after the allocation diagram. That way people know how much they're allocated, then we discuss their bill and who they should contact to discuss CMs